### PR TITLE
Updated MergeEvent

### PR DESCRIPTION
**Pull request by smit-modi**

### CODE REVIEW

File: event_webhook_types.go [hunk 1]
```diff
@@ -567,9 +569,13 @@ type MergeEvent struct {
 		Source                   *Repository  `json:"source"`
 		Target                   *Repository  `json:"target"`
 		HeadPipelineID           *int         `json:"head_pipeline_id"`
+		LastEditedAt             string       `json:"last_edited_at"`
+		LastEditedByID           int          `json:"last_edited_by_id"`
+		StateID                  StateID      `json:"state_id"`
```
Comment (svanharmelen) [inline event_webhook_types.go:574]: I prefer to use the order from the docs... Which means `LastEditedAt` and `LastEditedByID` should go above `MilestoneID` and `StateID` should go directly below `MilestoneID`...

File: event_webhook_types.go [hunk 2]
```diff
@@ -567,9 +569,13 @@ type MergeEvent struct {
...
 		LastCommit               struct {
 			ID        string     `json:"id"`
 			Message   string     `json:"message"`
+			Title     string     `json:"title"`
```
Comment (svanharmelen) [inline event_webhook_types.go:578]: I don't see title being mentioned in the docs.

File: event_webhook_types.go [hunk 3]
```diff
@@ -579,18 +585,19 @@ type MergeEvent struct {
 		} `json:"last_commit"`
 		BlockingDiscussionsResolved bool          `json:"blocking_discussions_resolved"`
 		WorkInProgress              bool          `json:"work_in_progress"`
+		TotalTimeSpent              int           `json:"total_time_spent"`
+		TimeChange                  int           `json:"time_change"`
+		HumanTotalTimeSpent         string        `json:"human_total_time_spent"`
+		HumanTimeChange             string        `json:"human_time_change"`
+		HumanTimeEstimate           string        `json:"human_time_estimate"`
```
Comment (svanharmelen) [inline event_webhook_types.go:592]: I don't see these fields in the docs for merge events, only for issue events...

File: event_webhook_types.go [hunk 4]
```diff
@@ -609,6 +616,18 @@ type MergeEvent struct {
...
+			Current  string `json:"current"`
+		} `json:"last_edited_at"`
+		LastEditedByID struct {
+			Previous int `json:"previous"`
+			Current  int `json:"current"`
+		} `json:"last_edited_by_id"`
+		MilestoneID struct {
+			Previous int `json:"previous"`
+			Current  int `json:"current"`
+		} `json:"milestone_id"`
```
Comment (svanharmelen) [inline event_webhook_types.go:630]: I don't see these fields in the docs for merge events, only for issue events...

**Review by svanharmelen — CHANGES_REQUESTED** [review]

File: event_webhook_types.go (hunk 1 above)
Comment (smit-modi) [reply event_webhook_types.go]: Fixed order

File: event_webhook_types.go (hunk 4 above)
Comment (svanharmelen) [reply event_webhook_types.go:630]: Cool, let me know when it's merged so I can have another look 👍🏻

Comment (smit-modi) [reply event_webhook_types.go:630]: @svanharmelen It is merged.

**Review by svanharmelen — APPROVED** [review]
LGTM 👍🏻  Thanks!

**Pull request merged**